src/simtools/job_execution/htcondor_script_generator.py

```diff
@@ -101,9 +100,20 @@ def _get_submit_script(args_dict):
     )
     core_scatter = args_dict["core_scatter"]
     core_scatter_string = f'"{core_scatter[0]} {core_scatter[1].to(u.m).value} m"'
+    view_cone = args_dict["view_cone"]
+    view_cone_string = f'"{view_cone[0].to(u.deg).value} deg {view_cone[1].to(u.deg).value} deg"'
```

Comment:
suggest to use builtin formatting 
view_cone_string = f'"{view_cone[0].to(u.deg):.2f} {view_cone[1].to(u.deg):.2f}"'

Reply:
Good point - implemented.